core/ordering/cosipbft/blocksync/default_test.go

```diff
@@ -169,46 +181,71 @@ func TestHandler_Stream(t *testing.T) {
 	err = handler.Stream(fake.Sender{}, fake.NewBadReceiver())
 	require.EqualError(t, err, "no announcement: receiver failed: fake error")
 
-	err = handler.Stream(fake.NewBadSender(), fake.NewReceiver(types.NewSyncMessage(6)))
+	handler.genesis = blockstore.NewGenesisStore()
+	err = handler.Stream(fake.Sender{}, fake.NewReceiver(msgs...))
+	require.EqualError(t, err, "reading genesis: missing genesis block")
+
+	handler.genesis.Set(otypes.Genesis{})
+	err = handler.Stream(fake.Sender{}, fake.NewReceiver(types.NewSyncMessage(fakeChain{err: xerrors.New("oops")})))
+	require.EqualError(t, err, "failed to verify chain: oops")
+
+	err = handler.Stream(fake.NewBadSender(), fake.NewReceiver(types.NewSyncMessage(makeChain(t, 6))))
 	require.EqualError(t, err, "sending request failed: fake error")
 
 	rcvr := fake.NewBadReceiver()
-	rcvr.Msg = []serde.Message{types.NewSyncMessage(6)}
+	rcvr.Msg = []serde.Message{types.NewSyncMessage(makeChain(t, 6))}
 	err = handler.Stream(fake.Sender{}, rcvr)
 	require.EqualError(t, err, "receiver failed: fake error")
 
-	msgs = []serde.Message{types.NewSyncMessage(6), msgs[1]}
+	msgs = []serde.Message{types.NewSyncMessage(makeChain(t, 6)), msgs[1]}
 	err = handler.Stream(fake.Sender{}, fake.NewReceiver(msgs...))
 	require.Error(t, err)
 	require.Regexp(t, "pbft catch up failed: mismatch link '[0]{8}' != '[0-9a-f]{8}'", err.Error())
 
-	err = handler.Stream(fake.NewBadSender(), fake.NewReceiver(types.NewSyncMessage(0)))
+	err = handler.Stream(fake.NewBadSender(), fake.NewReceiver(types.NewSyncMessage(makeChain(t, 0))))
 	require.EqualError(t, err, "sending ack failed: fake error")
 }
 
 // Utility functions -----------------------------------------------------------
```

Comment:
I don't mind changing to this, but then we should stay consistent with the rest, as we normally use

```
// -----------------------------------------------------------------------------
// Utility functions
```

Reply:
I replaced the occurrences.

Reply:
you put the line at the bottom, instead of top 😅

Reply:
oopsy !